### PR TITLE
add simplified Chinese translation

### PR DESCRIPTION
**Pull request by amorphobia**

### CODE REVIEW

File: Langs/0804.lang [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Chinese (Simplified) - China
+Translator=Xuesong Peng (@amorphobia)
+Version=2.2.5.0
+[Strings]
+Donate=捐赠
+Check for Updates=检查更新
+Your Windows 11 Compatibility Results are Below=您的 Windows 11 兼容性结果如下
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=现在访问 WhyNotWin11：https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=结果基于当前已知的要求！
```
Comment (walterlv) [inline Langs/0804.lang:10]:
```suggestion
Results Based on Currently Known Requirements!=结果基于当前已知的需求！
```

File: Langs/0804.lang [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Architecture (CPU + OS)=架构 (处理器 + 操作系统)
+Boot Method=启动方式
+CPU Compatibility=处理器兼容性
+CPU Core Count=处理器核心数
+CPU Frequency=处理器频率
+DirectX + WDDM2=DirectX + WDDM2
+Disk Partition Type=磁盘分区类型
+RAM Installed=已安装内存
+Secure Boot=安全启动
+Storage Available=可用储存
```
Comment (walterlv) [inline Langs/0804.lang:21]:
```suggestion
Storage Available=可用存储空间
```

File: Langs/0804.lang [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+64 Bit CPU=64 位处理器
+64 Bit OS=64 位操作系统
+32 Bit CPU=32 位处理器
+32 Bit OS=32 位操作系统
+Unable to Determine=无法判断
+Unable to Check List=无法检查列表
+Error Accessing List=访问错误列表
+Not Currently Listed as Compatible=当前未列为兼容
+Listed as Compatible=列为兼容
+Cores=核心
```
Comment (walterlv) [inline Langs/0804.lang:33]:
```suggestion
Cores=核
```

我觉得大家一般说“12 核 24 线程”，很少说“12核心，24 线程”。

File: Langs/0804.lang [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Threads=线程
+GPT Detected=检测到 GPT
+GPT Not Detected=未检测到 GPT
+Enabled=已启用
+Supported=支持
+Not Supported=不支持
+Disabled / Not Detected=已禁用 / 未检测到
+Drive(s) Meet Requirements=硬盘满足要求
```
Comment (walterlv) [inline Langs/0804.lang:41]:
```suggestion
Drive(s) Meet Requirements=硬盘满足需求
```

**Comment by micwoj92** [comment]
Closing as that language has already been submitted, feel free to submit patches later.

**Pull request closed without merging**